packages/core/src/transport/httpRequest.ts

```diff
@@ -73,3 +72,36 @@ function reportBeaconError(e: unknown) {
     addTelemetryError(e)
   }
 }
+
+export function fetchKeepAliveStrategy(
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:**if `fetch` `keepalive` just uses [`sendBeacon` under the hood](https://xgwang.me/posts/you-may-not-know-beacon/#fetch-keepalive), why do we not use `sendBeacon` as it has wider support?

Reply:
cf [previous step of this topic](https://github.com/DataDog/browser-sdk/pull/1701)

> In order to improve our data transfer, send request with fetch keepalive and fallback in order to later be able to access to intake response.